react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -172,14 +174,25 @@ class RCTAztecView: Aztec.TextView {
         recognizer.isEnabled = false
     }
 
-    // MARK - View Height: Match to content height
+    // MARK: - View height and width: Match to the content
 
     override func layoutSubviews() {
         super.layoutSubviews()
+        adjustWidth()
         fixLabelPositionForRTLLayout()
         updateContentSizeInRN()
     }
 
+    private func adjustWidth() {
```

Comment:
It seems very strange to me that we would need this. Can you explain why we need to adjust the width manually and it's not enough with flexbox?

Reply:
@dratwas could you please explain that?

Reply:
I couldn't make it work with `flexbox` only. I think it is because on initial render we have width set and if the text is bigger than that width it does the break line instead of expanding width. Additional thing is that the RN parent view doesn't know that the textView wants to expand until we send that information. However, if you have any idea how we could handle that in a different way, I would be more than grateful :)

Reply:
Just a quick update, I did some testing with a sample app and `TextInput` and what we want here seems well supported. I'm still not sure what exactly is preventing that from working in Gutenberg, but it should be possible in theory.

![RPReplay_Final1578911172](https://user-images.githubusercontent.com/8739/72248525-adf61000-35f7-11ea-97aa-39f038379dc6.gif)



```jsx
export default function App() {
  const [value, setValue] = React.useState( 'Button label' );
  return (
    <View style={ styles.container }>
      <TextInput
        style={ styles.input }
        value={ value }
        multiline={ true }
        onChangeText={ text => setValue( text ) }
      />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    alignContent: 'center',
  },
  input: {
    color: '#fff',
    backgroundColor: '#00f',
    fontSize: 24,
  }
});
```